src/DocumentDb/Microsoft.Omex.DocumentDb.csproj

```diff
@@ -16,13 +16,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="$(Microsoft_Azure_DocumentDB_Version)" />
-  </ItemGroup>
```

Comment:
You still have the core version defined within the props file. Is removing these if statements deliberate?

Reply:
Yes we don't need to have versions of this package.

Reply:
If that's the case, can you remove the second definition of it from the props file, i.e. the `System_Configuration_ConfigurationManager_Version_Core` one? If we're not using it anywhere, it doesn't seem like it would be relevant to have this anymore.

Reply:
@aszczepanski: I think you might have missed my response here

Reply:
Thanks, I missed that unused variable. It's removed now.

Reply:
Thanks Adam